keanu-project/src/main/java/io/improbable/keanu/distributions/continuous/StudentT.java

```diff
@@ -0,0 +1,97 @@
+package io.improbable.keanu.distributions.continuous;
+
+import java.util.Random;
+
+import static java.lang.Math.*;
+import static org.apache.commons.math3.special.Gamma.gamma;
+import static org.apache.commons.math3.special.Gamma.digamma;
+
+public class StudentT {
+	/**
+	 * Computer Generation of Statistical Distributions
+	 * by Richard Saucier
+	 * ARL-TR-2168 March 2000
+	 * 5.1.23 page 36
+	 */
+	public static double sample(double v, Random random) {
+		assert( v > 0. );
+		return Gaussian.sample( 0., 1., random) / sqrt( ChiSquared.sample((int) v, random) / v );
+		
+	}
+	
+	public static double pdf(double v, double t) {
+		double halfVPlusOne = (v + 1.) / 2.;
+		double halfV = v / 2.;
+		double numerator = gamma(halfVPlusOne);
+		double denominator = sqrt(v * PI) * gamma(halfV);
+		double multiplier = pow(1. + (pow(t, 2) / v), -halfVPlusOne);
+		
+		return (numerator / denominator) * multiplier;
+	}
+	
+	public static double logPdf(double v, double t) {
+		return log(pdf(v, t));
```

Comment:
Is there not a more simple function for log of the pdf? Are there any problems, numerical stability-wise, when t is really big?